airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/MssqlSourceTest.java

```diff
@@ -106,6 +114,43 @@ void testDiscoverWithPk() throws Exception {
     assertEquals(CATALOG, actual);
   }
 
+  @Test
+  public void testTableWithNullCursorValueShouldThrowException() throws Exception {
```

Comment:
One comment I got on the postgres test was to check also with a View

Reply:
updated with a view test